src/shogun2-util/src/main/java/de/terrestris/shogun2/util/interceptor/WcsRequestInterceptorInterface.java

```diff
@@ -0,0 +1,14 @@
+package de.terrestris.shogun2.util.interceptor;
+
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
Will this actiually be enforced, that implementations of this interface must also be a `@Component`? I didn't know that.